src/main/java/com/iexec/common/docker/DockerRunRequest.java

```diff
@@ -72,30 +73,37 @@ public String getStringArgsCmd() {
         return devices != null ? new ArrayList<>(devices) : List.of();
     }
 
-    // override builder's isSgx() & devices() methods
+    public SgxDriverMode getSgxDriverMode() {
+        return sgxDriverMode != null ? sgxDriverMode : SgxDriverMode.NONE;
```

Comment:
If `sgxDriverMode` is null, should we store `SgxDriverMode.NONE` in `sgxDriverMode` ?

Reply:
We can do so.

Reply:
Done in https://github.com/iExecBlockchainComputing/iexec-common/pull/308/commits/24b0c04479f5ce6a5963a0c09882a81f478e7c41.